README.rst

```diff
@@ -5,6 +5,16 @@ frontend-enterprise
 
 frontend-enterprise contains utility functions for supporting enterprise features.
 
+Dependency notes
+-----
+
+v6 and higher of query-string will fail the es5 check
+eslint-plugin-import needed at least 2.22.1 to avoid failures with an infinity symbol (see https://stackoverflow.com/questions/64790681/eslint-error-configuration-for-rule-import-no-cycle-is-invalid)
```

Comment:
I will also likely update this version in frontend-build, ran this idea past @davidjoy  already but having the dep in our package.json unblocks this PR

Reply:
@binodpant I can look into upgrading eslint-plugin-import in frontend-build, or review any PRs for it. I'm overdue to look at some of the upgrade PRs for frontend-build as the primary technical "owner" :)